docs/pages/docs/typescript.en-US.mdx

```diff
@@ -60,7 +60,6 @@ The following hooks support type inference when you add a const assertion to `ab
 - [useContractRead](/docs/hooks/useContractRead)
 - [useContractReads](/docs/hooks/useContractReads)
 - [useContractWrite](/docs/hooks/useContractWrite)
-- [usePrepareContractWrite](/docs/hooks/usePrepareContractWrite)
```

Comment:
Let's not remove this line. Instead, can we link it to `/docs/prepare-hooks/usePrepareContractWrite`?

Reply:
Aha, sorry, was looking among the hooks folder and guessed it was removed. Will fix! Thanks!